packages/core/src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -254,6 +254,26 @@ export default class BraintreeHostedForm {
         }
     }
 
+    private _mapErrors(fields: any): any {
+        const errors = {};
+
+        if (fields) {
+            for (const [key, value] of Object.entries(fields)) {
+                const { isValid, isEmpty, isPotentialyValid } = value as any;
+
+                (errors as any)[key] = {
+                    [key]: {
```

Comment:
I am thinking maybe we don't require an additional `[key]` layer here. Since the current format is 
```typescript
errors: {
        cvv: {
            cvv: {
                isEmpty: true,
                isPotentialyValid: true,
                isValid: false,
            },
        },
    },
```
Without it, it will be looks like 
```typescript
errors: {
      cvv: {
          isEmpty: true,
          isPotentialyValid: true,
          isValid: false,
      },
 },
```
Which is less repetitive.

Reply:
Good thinking 😄

Reply:
Let's try to use the second option, because it looks much better than prev one

Reply:
Let me update this. 👍